app/models/organization.rb

```diff
@@ -43,9 +43,16 @@ def logo_delete
     @logo_delete || '0'
   end
 
+  def set_github_org
+    if self.projects.first
+      self.github_org = Project.parse_git_url(self.projects.first.submitted_github_url).split('/').first
```

Comment:
Line is too long. [104/80]<br>Redundant `self` detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.